### PR TITLE
HTTP binary sensor and sensor docs

### PR DESCRIPTION
**Pull request by fabaff**
This PR contains some more details about the usage of the HA REST API. More information for sensors and binary sensors are provided.

Till now the only place to find information about this feature was embedded in the API docs. I hope that this could make the API more visible and stop me for doing stuff that is already there.

### CODE REVIEW

**Comment by fabaff** [comment]
I'm open for suggestions and feedback :-D

**Comment by fabaff** [comment]
I would be nice if someone could verify the stuff I wrote. Just be sure that it's complete rubbish. Thanks.

**Comment by balloob** [comment]
It looks good.

I think we should somehow be able to flag each page that we put out as a platform/component that is actually just a tutorial to use some other part of Home Assistant. There is `sensor.http`, `binary_sensor.http`, `notify.mqtt` and `ifttt.manything`.

Please be aware that this works for the `sensor` and `binary_sensor` platforms as the components do not provide any services to interact with the entities. This will not work for a light because services as `light.turn_on` etc will not work.

File: source/_components/binary_sensor.http.markdown [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+---
+
+The URL for a binary sensor looks like the example below:
+
+```bash
+http://IP_ADDRESS:8123/api/states/binary_sensor.DEVICE_NAME
+```
+
+<p class='note'>
+It's suggested that you choose an unique device name (DEVICE_NAME) to avoid clashes with other devices.
```
Comment (balloob) [inline source/_components/binary_sensor.http.markdown]: It's not only suggested, it's necessary. "You should choose a unique…"

**Comment by balloob** [comment]
You can merge it after fixing that 1 sentence. :dolphin:

**Comment by fabaff** [comment]
> I think we should somehow be able to flag each page that we put out as a platform/component that is actually just a tutorial to use some other part of Home Assistant. There is sensor.http, binary_sensor.http, notify.mqtt and ifttt.manything.

Agreed. I started to realize while creating the MQTT notification page that this kind of documentation doesn't fit in the existing categories. Due to the lack of better ideas I just placed them in the category which fits best from my point of view.

**Comment by balloob** [comment]
I think that is fine. That is the place people will look for it.

**Pull request merged**